app/helpers/audits_helper.rb

```diff
@@ -15,8 +15,17 @@ def id_to_label(name, change, truncate = true)
           label = name.classify.gsub('Id', '').constantize.find(change).to_label
         rescue NameError
           # fallback to the value only instead of N/A that is in generic rescue below
-          return change.to_s
+          return _("Missing(ID: %s)") % change
         end
+      when /.*_ids$/
+        existing = name.sub('_ids', '').classify.constantize.where(id: change).index_by(&:id)
```

Comment:
what if foreign key does not map to class name? since the key in `audit_association` uses relation name, this might be more reliable

```
audit.auditable_type.constantize.reflect_on_association(name.sub('_ids', '')).klass.where(id: change).index_by(&:id)
```

also using following regexp with sub would be slightly safer
```
name.sub(/_ids$/, '')
```